arkouda/dataframe.py

```diff
@@ -382,25 +382,83 @@ def _repr_html_(self):
     def _ipython_key_completions_(self):
         return self._columns
 
-    def drop(self, keys):
+    def _drop_column(self, keys):
         """
         Drop a column or columns from the dataframe, in-place.
 
-        Parameters
-        ----------
-        keys : str or list
-            The column(s) to be dropped.
+        keys : list
+            The labels to be dropped on the given axis
         """
-
-        if isinstance(keys, str):
-            keys = [keys]
         for key in keys:
             # Do not allow the user to drop the index column
             if key == 'index':
                 raise KeyError('The index column may be reset, but not dropped.')
 
             del self[key]
 
+    def _drop_row(self, keys):
+        """
+        Drop a row or rows from the dataframe, in-place.
+
+        keys : list
+            The indexes to be dropped on the given axis
+        """
```

Comment:
i think we should allow for users to pass in a pdarray for keys in addition to a list, but that can be handled in a different PR